.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,14 @@
+- &hook
+  id: nbdev_clean
```

Comment:
Here's what these mean:

- `language: system` – run `entry` as a shell command. You can also use `language: python` which would install nbdev in its own virtual environment. Since we expect the user to have nbdev installed already, I opted for `system`
- `always_run: true` – always run these hooks regardless of which files were committed. We could also configure so that it only runs when jupyter and/or python files change. I thought it's fast enough that this is a good default
- `pass_filenames: false` – run `entry` as is. If this is `true`, then its called like `nbdev_clean file1 file2 file3` for each staged file. Our commands check all files if run without args, which I thought was a good default, since it matches our current checks in CI

Reply:
`language: system` also makes it **much** easier during development, since it will use the current environment's nbdev, fastcore, ghapi, etc